config.xml

```diff
@@ -77,10 +77,16 @@
     <splash src="resources/ios/splash/Default@2x~iphone.png" width="640" height="960"/>
     <splash src="resources/ios/splash/Default~iphone.png" width="320" height="480"/>
   </platform>
+  <platform name="wp8">
```

Comment:
well, but my concern is also... what does wp8 come from? Has anything to do with [Window Phone 8](https://forum.ionicframework.com/t/does-ionic-supports-windows-phone-8/29233/3)?